### ISSUE TITLE
Keyboard navigation issues with Markdown Editors

### ISSUE THREAD

**Issue by jobara** [laravel-filament/filament#17854]
### Package

filament/filament

### Package Version

v4

### Laravel Version

v12

### Livewire Version

v3

### PHP Version

PHP 8.4

### Problem description

When there are adjacent Markdown Editors in a form, you are unable to use the `tab` key to navigate with the keyboard from one markdown editor to the next. This results in a keyboard trap.

Additionally using the `shift-tab` keyboard combination from within a markdown editor always moves focus back to the first focusable element in the form instead of to the previous one.

### Expected behavior

Using the `tab` should move to the next focusable element. Using the `shift-tab` keyboard combination should move to the previous focusable element.

### Steps to reproduce

1. Open a form that that has two or more adjacent Markdown Editors. (see reproduction repository for example)
2. Place focus into the first Markdown editor
3. Without selecting any text, press the `tab` key on the keyboard to attempt to move focus to the next Markdown Editor
4. Notice that focus remains in the current Markdown Editor
5. Using the keyboard press `shift-tab` keyboard combination
6. Notice that focus is moved to the first focusable element in the form instead of the previous one.

### Reproduction repository (issue will be closed if this is not valid)

https://github.com/jobara/filament-markdown-editor-keyboard-trap

### Relevant log output

```shell

```

**Comment by jobara** [comment]
I assume the issue is related to `moveToNextField` and `moveToPreviousField`.

https://github.com/filamentphp/filament/blob/8d6f6ef076cc0ca618908d4d1f4235e7402c1c92/packages/forms/resources/js/components/markdown-editor/EasyMDE.js#L2343-L2377

**Comment by danharrin** [comment]
Any thoughts on this reproduction repository @alex-elivate?

**Comment by alex-elivate** [comment]
I was able to duplicate this.  I got stuck here.  Fun puzzle, thanks! I'm on it.

<img width="1295" height="1006" alt="Image" src="https://github.com/user-attachments/assets/99847ee8-cf66-4958-91b9-30ca4a58ea81" />

**Comment by jobara** [comment]
@alex-elivate thank you for fixing this! It's very much appreciated.
